spec/graphql/mutations/request_password_recovery_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+describe Mutations::RequestPasswordRecovery do
+  let(:response) { ApplicationSchema.execute(query, {}).as_json }
+  let(:user) { create :user }
+  let(:query) do
+    <<-GRAPHQL
+      mutation {
+        requestPasswordRecovery(email: "#{user.email}") {
```

Comment:
Would it be better to use explicit input?
```suggestion
        requestPasswordRecovery(email: "user@example.com") {
```
and then in one spec example create a user with this email.